Modix.Bot/Behaviors/PromotionPollBehavior/CachedPromoPoll.cs

```diff
@@ -0,0 +1,17 @@
+using Discord;
+using Modix.Data.Models.Promotions;
+
+namespace Modix.Bot.Behaviors
+{
+    public class CachedPromoPoll
+    {
+        public IUserMessage Message { get; set; }
```

Comment:
Caching this is a really really bad idea. These objects are not immutable and you don't control their lifetime. Pull this from the `IDiscordSocketClient` each time it's needed.